appveyor.yml

```diff
@@ -44,10 +44,18 @@ before_build:
   - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
   - nuget restore
 
+
+after_build:
+  - cd src\ConsoleApp\bin\%CONFIGURATION%
+  - 7z a package.zip *
```

Comment:
This should probably be only some of the files. We don't need the PDB files or many of the XML files (we will need OSPSuite.Dimensions.xml and maybe OSPSuite.PKParameters.xml. I don't think we need .config files either if they are there. This line probably zips up the .exe file too which we don't need.

Reply:
Let's name the package something like `pk-sim-r-dependencies.zip`